.github/workflows/ci.yml

```diff
@@ -1794,7 +1794,7 @@ jobs:
         run: chmod +x ./uv
 
       - name: "Configure AWS credentials"
-        uses: aws-actions/configure-aws-credentials@351d894493fd3289754a5471c0892ba92fa0abe2
+        uses: aws-actions/configure-aws-credentials@351d894493fd3289754a5471c0892ba92fa0abe2 # v5.0.0
```

Comment:
I don't think this is the right commit for v5.0.0 

https://github.com/aws-actions/configure-aws-credentials/commit/a03048d87541d1d9fcf2ecf528a4a65ba9bd7838